sonaranalyzer-dotnet/Directory.Build.props

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project>
+
+  <PropertyGroup Condition="'$(Configuration)' == 'Debug'">
+    <DebugType>full</DebugType>
+    <DebugSymbols>true</DebugSymbols>
+    <OutputPath>bin\Classic\Debug\</OutputPath>
+    <IntermediateOutputPath>obj\Classic</IntermediateOutputPath>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(Configuration)' == 'Release'">
+    <DebugType>pdbonly</DebugType>
```

Comment:
Wasn't this previously set to _Full_ so that we got code coverage in the CIX builds? Is it no longer required?

Reply:
I don't know this seems to be done this way by most of roslyn analyzers to I wanted to give it a try.